api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -44,59 +46,82 @@
   private String username;
   // The email address that can be used to contact the user.
   private String contactEmail;
-  private String givenName;
-  private String familyName;
-  private String phoneNumber;
-  private String professionalUrl;
-  private String currentPosition;
-  private String organization;
   private Double freeTierCreditsLimitDollarsOverride = null;
-  private Short freeTierCreditsLimitDaysOverride = null;
-  private Timestamp lastFreeTierCreditsTimeCheck;
   private Timestamp firstSignInTime;
-  private Timestamp firstRegistrationCompletionTime;
   private Set<Short> authorities = new HashSet<>();
-  private Boolean idVerificationIsValid;
```

Comment:
we don't do ID verification.  We will again soon, but that will be handled by RAS+IAL2, not here.